bot/cogs/snakes.py

```diff
@@ -1,11 +1,10 @@
-# coding=utf-8
 import logging
 from typing import Any, Dict
 
 from discord.ext.commands import AutoShardedBot, Context, command
 
 log = logging.getLogger(__name__)
-
+import discord
```

Comment:
This module level import should be at the top of the file.

Reply:
(preferably before `from discord.ext.commands import AutoShardedBot, Context, command`)